common/persistence/domainReplicationQueue.go

```diff
@@ -125,58 +122,6 @@ func (q *domainReplicationQueueImpl) GetAckLevels() (map[string]int, error) {
 	return q.queue.GetAckLevels()
 }
 
-func (q *domainReplicationQueueImpl) purgeAckedMessages() error {
-	ackLevelByCluster, err := q.GetAckLevels()
-	if err != nil {
-		return fmt.Errorf("failed to purge messages: %v", err)
-	}
-
-	if len(ackLevelByCluster) == 0 {
-		return nil
-	}
-
-	minAckLevel := math.MaxInt64
-	for _, ackLevel := range ackLevelByCluster {
-		if ackLevel < minAckLevel {
-			minAckLevel = ackLevel
-		}
-	}
-
-	err = q.queue.DeleteMessagesBefore(minAckLevel)
-	if err != nil {
-		return fmt.Errorf("failed to purge messages: %v", err)
-	}
-
-	q.metricsClient.
-		Scope(metrics.FrontendDomainReplicationQueueScope).
-		UpdateGauge(metrics.DomainReplicationTaskAckLevel, float64(minAckLevel))
-
-	q.ackLevelUpdated = false
-
-	return nil
-}
-
-func (q *domainReplicationQueueImpl) purgeProcessor() {
-	ticker := time.NewTicker(purgeInterval)
-	defer ticker.Stop()
-
-	for {
-		select {
-		case <-q.done:
-			return
-		case <-ticker.C:
-			if q.ackLevelUpdated {
-				err := q.purgeAckedMessages()
-				if err != nil {
-					q.logger.Warn("Failed to purge acked domain replication messages.", tag.Error(err))
-				}
-			}
-		case <-q.ackNotificationChan:
-			q.ackLevelUpdated = true
-		}
-	}
-}
-
```

Comment:
why deleting this 2 functions?
@yux0 are these 2 functions still useful or not?

Reply:
Thanks for raising the concern. I will add those functions back